core/src/main/scala/org/http4s/headers/Accept.scala

```diff
@@ -1,12 +1,25 @@
 package org.http4s
 package headers
 
+import org.http4s.util.{Renderable, Writer}
+
 import scalaz.NonEmptyList
 
-object Accept extends HeaderKey.Internal[Accept] with HeaderKey.Recurring
+object Accept extends HeaderKey.Internal[Accept] with HeaderKey.Recurring {
+  implicit def withDefaultQValue(mediaRange: MediaRange): MediaRangeAndQValue =
```

Comment:
Without this, either all the constants in `MediaType` need to exist on `MediaRangeAndQValue`, or else the conversion is explicit:

```scala
Accept(`text/xml`.withDefaultQValue)
```